### DISCUSSION TITLE
Does NeMo SSL models support in Riva?

### DISCUSSION THREAD

**Discussion by mehadi92** [NVIDIA-NeMo/NeMo#7363] [Q&A]
I know the conformer model supports in Riva but does the SSL conformer model support Riva? For example, can I use this model in Riva? model: https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/ssl_en_conformer_large

Thanks

**Comment by titu1994** [comment]
You can use it for finetuning on ASR dataset. SSL checkpoints cannot be used as is for inference

**Comment by mehadi92** [comment]
@titu1994 Does the finetune model can be used in the Riva?

For example, In this [notebook](https://github.com/NVIDIA/NeMo/blob/main/tutorials/asr/Self_Supervised_Pre_Training.ipynb) first pre-training with SSL after that SSL model was used for fine-tuning. Can I use the final fine-tuned model in the Riva?

Thanks
